test/clj_diff/test/core.clj

```diff
@@ -32,7 +32,7 @@
 
 (deftest patch-test
   (is (= (patch "aba"
-                {:+ [[1 \c]], :- [1]})
+                {:+ [[0 \c]], :- [1]})
```

Comment:
For my own understanding, why change the test? Both `[0 \c]` `[1 \c]` do `patch` to `"aca"`. 

Removing the `:-` and only patching with the `:+` we get:
```clj
  (patch "aba" {:+ [[0 \c]]}) ;; "acba"
  (patch "aba" {:+ [[1 \c]]}) ;; "abca"
```

At least conceptually `[[1 \c]]` gets the intention across better as you want to replace the character at index 1. If I want to replace the character at index 0, then I'd do `{:+ [[0 \c]] :-[0]}`.

Reply:
yes, indeed.

Reply:
done